priv/cabbage/Makefile

```diff
@@ -14,6 +14,12 @@ start_daemon_services:
 	cd priv/cabbage/ && \
 	docker-compose -f ../../docker-compose.yml -f docker-compose-cabbage.yml up -d
 
+start_daemon_services_reorg:
+	cd ../../ && \
+	make init_test_reorg && \
+	cd priv/cabbage/ && \
+	docker-compose -f ../../docker-compose-reorg.yml -f docker-compose-cabbage.yml up -d
```

Comment:
Are we using `docker-compose-reorg.yml` much outside of the `cabbage` repo/context? I've only noticed one instant where [we use it to output logs for CI](https://github.com/omgnetwork/elixir-omg/pull/1579/files#diff-1d37e48f9ceff6d8030570cd36286a61R584). Perhaps we move it into this repo?

Reply:
I move it into the `cabbage` directory